docs/source/API/core/parallel-dispatch/ParallelForTag.rst

```diff
@@ -0,0 +1,50 @@
+``ParallelForTag()``
+====================
+
+.. role::cpp(code)
+    :language: cpp
+
+Header File: ``<Kokkos_ExecPolicy.hpp>``
+
+.. _text: ../parallel-dispatch/parallel_for.html
+
+.. |text| replace:: ``parallel_for``
+
+A tag used in team size calculation functions to indicate that the functor for which a team size is being requested is being used in a |text|_
```

Comment:
Same as before, probably easier to cross reference https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#role-cpp-func

Reply:
done